src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -230,6 +230,7 @@ private async Task GenerateDeploymentRecipeSnapShot(Recommendation recommendatio
             recipe.Name = recipeName;
             recipe.CdkProjectTemplateId = null;
             recipe.CdkProjectTemplate = null;
+            recipe.CreatedFromSavedDeploymentProject = true;
```

Comment:
I know you change the name of this field due to a my previous comment but I didn't realize at the time it was being persisted back into the recipe file. In the recipe file the name is very long and doesn't really fit in compared to the other fields. How about `PersistedDeploymentProject`?